isis/tests/FunctionalTestsJigsaw.cpp

```diff
@@ -1194,6 +1194,339 @@ End)");
   EXPECT_NEAR(columns[3].toDouble(), 1699.84329956, 0.0001);
   EXPECT_PRED_FORMAT2(AssertQStringsEqual, columns[4], "FREE");
   EXPECT_NEAR(columns[5].toDouble(), 5.34723296, 0.0001);
+}
+
+
+TEST_F(ApolloNetwork, FunctionalTestJigsawScconfig) {
+  QTemporaryDir prefix;
+  QString outCnetFileName = prefix.path() + "/outTemp.net";
+  QString scconfigPath = prefix.path() + "/scconfig.pvl";
+
+  std::istringstream scconfigStr(R"(Object = SensorParameters
+    Group = APOLLO15/METRIC
+      CKDegree=2
+      CKSolveDegree=2
+      CamSolve=Angles
+      Twist=yes
+      SPSolve=positions
+      Spacecraft_position_sigma=1000.0
+      Camera_Angles_Sigma=2.0
+    EndGroup
+    Group = "LUNARRECONNAISSANCEORBITER/NACR"
+      CamSolve=accelerations
+      Twist=yes
+      OverExisting=yes
+      SPSolve=accelerations
+      overhermite=yes
+      Spacecraft_position_sigma=100.0
+      Spacecraft_velocity_sigma=1.0
+      Spacecraft_acceleration_sigma=0.1
+      Camera_Angles_Sigma=2.0
+      Camera_AngularVelocity_Sigma=0.1
+      Camera_AngularAcceleration_Sigma=0.01
+    EndGroup
+    Group = "LUNARORBITER5/HIGHRESOLUTIONCAMERA"
+      CamSolve=angles
+      Twist=yes
+      Camera_Angles_Sigma=2.0
+      SPSolve=positions
+      Spacecraft_position_sigma=1000.0
+    EndGroup
+  EndObject
+  End)");
+
+  Pvl scconfig;
+  scconfigStr >> scconfig;
+  scconfig.write(scconfigPath);
+
+  QVector<QString> args = {"fromlist="+cubeListFile, "cnet="+controlNetPath,
+                           "onet="+outCnetFileName, "scconfig="+scconfigPath,
+                           "radius=true", "point_radius_sigma=50",
+                           "bundleout_txt=no", "output_csv=no",
+                           "residuals_csv=no", "file_prefix="+prefix.path()+"/scconfig"};
+
+   UserInterface options(APP_XML, args);
+
+   try {
+     jigsaw(options);
+   }
+   catch (IException &e) {
+     FAIL() << "Failed to bundle: " << e.what() << std::endl;
+   }
+
+   CSVReader header = CSVReader(prefix.path()+"/scconfig_bundleout_images.csv",
```

Comment:
I'm not sure why this file is not bundleout_images_APOLLO_METRIC.csv with the new changes. Might be due to using the Apollo Network fixture or needing to make changes in my scconfig file?

Reply:
This is because the apollo network has only apollo metric images in it. I think we can combine the observation LRONAC images and the Apollo images to make a combined network. I will tinker with this a little bit.

Reply:
Would it be an option to instead use the ObservationPair fixture since it will output a NACL.csv and a NACR.csv?

Reply:
No, we want a network were we want to solve for different parameters on different sensors. The original test was a framing camera and a line scan camera.